frontend/packages/topology/integration-tests/package.json

```diff
@@ -5,5 +5,9 @@
   "private": true,
   "cypress-cucumber-preprocessor": {
     "step_definitions": "support/step-definitions/*/"
+  },
+  "scripts": {
+    "test-cypress": "../../../node_modules/.bin/cypress open --env openshift=true",
+    "test-cypress-headless": "node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless cypress:cli,cypress:server:specs --spec \"features/pipelines/*.feature\";"
```

Comment:
why we have `features/pipelines/*.feature` in the topology integration test package?

Reply:
Updated